ckb-integration-test/src/case/rfc0028/v2019.rs

```diff
@@ -0,0 +1,207 @@
+use super::{ERROR_IMMATURE, PASS};
+use crate::case::{Case, CaseOptions};
+use crate::{CKB2019, CKB2021};
+use ckb_testkit::util::since_from_relative_timestamp;
+use ckb_testkit::NodeOptions;
+use ckb_testkit::{BuildInstruction, Nodes};
+use ckb_types::core::Capacity;
+use ckb_types::packed::OutPoint;
+use ckb_types::{
+    core::TransactionBuilder,
+    packed::{CellInput, CellOutput},
+    prelude::*,
+};
+use std::collections::HashSet;
+
+pub struct RFC0028V2019;
+
+impl Case for RFC0028V2019 {
+    fn case_options(&self) -> CaseOptions {
+        CaseOptions {
+            make_all_nodes_connected: false,
+            make_all_nodes_synced: false,
+            make_all_nodes_connected_and_synced: false,
+            node_options: vec![
+                NodeOptions {
+                    node_name: String::from("node2019"),
+                    ckb_binary: CKB2019.read().unwrap().clone(),
+                    initial_database: "testdata/db/Epoch2V1TestData",
+                    chain_spec: "testdata/spec/ckb2019",
+                    app_config: "testdata/config/ckb2019",
+                },
+                NodeOptions {
+                    node_name: String::from("node2021"),
+                    ckb_binary: CKB2021.read().unwrap().clone(),
+                    initial_database: "testdata/db/Epoch2V2TestData",
+                    chain_spec: "testdata/spec/ckb2021",
+                    app_config: "testdata/config/ckb2021",
+                },
+            ]
+            .into_iter()
+            .collect(),
+        }
+    }
+
+    fn run(&self, nodes: Nodes) {
+        let node2019 = nodes.get_node("node2019");
+        let node2021 = nodes.get_node("node2021");
+
+        node2021.mine(1);
+        node2019
+            .pull_node(node2021)
+            .expect("node2019 pull node2021");
+        let median_time_block_count = node2021.consensus().median_time_block_count.value();
+        let t = node2021.get_tip_block().timestamp();
+        let old_tip_number = node2021.get_tip_block_number();
+
+        // [(relative_millis, input_median_time, input_committed_time, tip_median_time, expected_result)]
+        let cases = vec![
+            (1000, t + 2000, t + 3000, t + 5000, PASS),
+            (1000, t + 2000, t + 3000, t + 4000, PASS),
```

Comment:
Why not using the boundary case?

```suggestion
            (1000, t + 2000, t + 3000, t + 3000, PASS),
```